GitUI/Avatars/AvatarService.cs

```diff
@@ -122,5 +122,10 @@ SafetynetAvatarProvider mainProvider
 
             return (mainProvider, cacheCleaner);
         }
+
+        public static void UpdateAvatarInitialFontsSettings()
```

Comment:
Why is this static? And what's the purpose of this?

Reply:
It is static because everything is static in this static class.
The purpose is to access the instance of InitialsAvatarProvider defined in this class to reset cached font when the setting is changed.